egui/src/widgets/plotting/plot.rs

```diff
@@ -419,18 +171,59 @@ impl Plot {
         self.show_y = show_y;
         self
     }
+
+    /// Always keep the x-axis centered.
```

Comment:
Please document what the default is for this and the other methods (I know I hadn't, but better late than never)